usersimcrs/simulator/agenda_based/interaction_model.py

```diff
@@ -16,22 +16,16 @@
 class InteractionModel:
     """Represents an interaction model."""
 
-    # TODO: These should probably be loaded from the interaction model yaml file.
-    # We can require that these are defined.
-    INTENT_START = Intent("DISCLOSE.NON-DISCLOSE")
-    INTENT_STOP = Intent("STOP")
-    # TODO: Add intents for item consumption and preferences
-    # See: https://github.com/iai-group/UserSimCRS/issues/111
-    # The user expresses that they have consumed an item in the past.
-    INTENT_ITEM_CONSUMED = Intent("NOTE.YES")
-    # The user responds to a recommendation positively or negatively.
-    INTENT_LIKE = Intent("NOTE.LIKE")
-    INTENT_DISLIKE = Intent("NOTE.DISLIKE")
-    INTENT_NEUTRAL = Intent("ADD-INTENT-NAME")  # TODO: intent to be created
-    # The user expresses preference on some slot/value.
-    INTENT_DISCLOSE = Intent("DISCLOSE")
-    # The user cannot answer.
-    INTENT_DONT_KNOW = Intent("ADD-INTENT-NAME")  # TODO: intent to be created
+    REQUIRED_INTENTS = {
```

Comment:
Can you add a comment explaining what this is for?

Reply:
Done.